.github/workflows/node.js.yml

```diff
@@ -87,7 +87,10 @@ jobs:
           cache: 'npm'
 
       - name: Build
-        run: npm ci
+        run: |
+          npm ci
+          cd examples/aws-lambda-typescript
+          npm ci
```

Comment:
Why is this in the CI? Are we running tests in the example folder?

Reply:
No. But I enabled eslint in examples folder to make sure all codebase conforms to it. Since this is a typescript project, eslint was trying to check types and it kept failing because it couldn't find any modules installed.